app/Http/Controllers/AboutController.php

```diff
@@ -103,7 +103,10 @@ public function index(Request $request)
             'stat_services' => Service::count(),
             'stat_slas' => Sla::count(),
             'stat_storage' => Storage::count(),
-            'stat_syslog' => Syslog::count(),
+            'stat_syslog' => DB::table('information_schema.tables')
```

Comment:
This only works on mysql, we want to avoid making database specific implementations